src/order/locally_finite.lean

```diff
@@ -529,31 +529,31 @@ instance : locally_finite_order (with_top α) :=
   finset_mem_Ico := λ a b x, match a, b, x with
     |       ⊤,       b,       x := iff_of_false (not_mem_empty _)
                                      (λ h, not_top_lt $ h.1.trans_lt h.2)
-    | (a : α),       ⊤,       ⊤ := by simp [with_top.locally_finite_order._match_3]
-    | (a : α),       ⊤, (x : α) := by simp [with_top.locally_finite_order._match_3, coe_eq_coe,
+    | (a : α),       ⊤,       ⊤ := by simp [with_top.locally_finite_order._match_2]
+    | (a : α),       ⊤, (x : α) := by simp [with_top.locally_finite_order._match_2, coe_eq_coe,
```

Comment:
The `match` numbers changed without any apparent reason, which is not looking looking good for the maintainability of the code 😕

Reply:
I wouldn't worry too much, lean4 should fix this